cms/static/sass/_build.scss

```diff
@@ -67,6 +67,7 @@
 @import 'views/group-configuration';
 @import 'views/video-upload';
 @import 'views/certificates';
+@import 'views/creative-commons';
```

Comment:
I don't see a `cms/static/sass/components` directory. Where should I put the `_creative-commons.scss` file?